stacks/api-stack.js

```diff
@@ -1,12 +1,61 @@
-import { Api } from '@serverless-stack/resources'
+import {
+  Api,
+  Bucket,
+  Table
+} from '@serverless-stack/resources'
+
+import { getConfig } from './config.js'
 
 /**
  * @param {import('@serverless-stack/resources').StackContext} properties
  */
 export function ApiStack({ stack }) {
+  // @ts-ignore "prod" | "dev" | "staging" only allowed for stage
+  const stackConfig = getConfig(stack.stage)
+
+  /**
+   * This table takes a stored CAR and makes an entry in the store table
+   * to associate the uploaders DID with a payload CID.
+   * You can also optionally indicate an application DID, origin and size
+   */
+   const storeTable = new Table(stack, 'store_table', {
+    fields: {
+      uploaderDID: 'string',
+      payloadCID: 'string',
+      applicationDID: 'string',
+      origin: 'string',
+      size: 'number',
```

Comment:
is it size in bytes?

Reply:
Yes, see https://github.com/web3-storage/w3protocol/pull/160/files